cypress/e2e/layers.cy.ts

```diff
@@ -295,13 +306,14 @@ describe("layers", () => {
           when.click("layer-list-item:background:" + bgId);
 
           let errorSelector = ".CodeMirror-lint-marker-error";
-          should.notExist(errorSelector);
+          // Not a good idea to check that something doesn't exist. infinite things don't exist.
```

Comment:
This checks that the same element does not exist before the click, and does exist after the click.
If you have a better way to test this please suggest, otherwise I believe this comment can be removed as I don't think it's actionable...